recipes/taglib/all/patches/1.12-0001-fix-error-duplicate-volatile.patch

```diff
@@ -0,0 +1,22 @@
+From eecfd3eeed5c98c0e0c85d7e0d093d3253ae43fe Mon Sep 17 00:00:00 2001
+From: Oleg Antonyan <oleg.b.antonyan@gmail.com>
+Date: Sat, 8 May 2021 21:16:58 +0200
+Subject: [PATCH] fix error: duplicate volatile
+
+---
+ taglib/toolkit/trefcounter.cpp | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/taglib/toolkit/trefcounter.cpp b/taglib/toolkit/trefcounter.cpp
+index 18cb596c0..ae9f30935 100644
+--- a/taglib/toolkit/trefcounter.cpp
++++ b/taglib/toolkit/trefcounter.cpp
+@@ -66,7 +66,7 @@ namespace TagLib
+     RefCounterPrivate() :
+       refCount(1) {}
+ 
+-    volatile ATOMIC_INT refCount;
++    ATOMIC_INT refCount;
+   };
+ 
+   RefCounter::RefCounter() :
```

Comment:
Actually they have merged https://github.com/taglib/taglib/pull/1012/commits/f9f3894c861445eb5f0ed5deeb1b12a1bc50513e

Reply:
Yes, after the 1.12 release

Reply:
I don't understand, what does it change? https://github.com/taglib/taglib/commit/1d24bd3399f2284104cbe87d017f87325b1797ab seems to be the proper fix.

Reply:
https://github.com/taglib/taglib/commit/1d24bd3399f2284104cbe87d017f87325b1797ab was committed on May 13.
The 1.12 release happened on Feb 16 (https://github.com/taglib/taglib/releases/tag/v1.12)

Reply:
Sure, but I still don't understand the patch.

Reply:
Oh, indeed.
https://github.com/taglib/taglib/commit/1d24bd3399f2284104cbe87d017f87325b1797ab.patch should be added instead.